qiskit/providers/models/backendconfiguration.py

```diff
@@ -329,6 +332,12 @@ def __init__(self,
         self.discriminators = discriminators
         self.hamiltonian = hamiltonian
 
+        # only raise dt/dtm warning once
+        if not PulseBackendConfiguration._dt_warning_done:
+            warnings.warn('`dt` and `dtm` now have units of seconds(s) rather '
+                          'than nanoseconds(ns).', DeprecationWarning)
```

Comment:
I'm not sure I'd consider this a deprecation warning. Deprecation warnings are there to tell users the thing you just used will be changing or going away in the future. While this is the other way the thing you just used has different behavior from previous releases. Looking at: https://docs.python.org/3/library/warnings.html#warning-categories I think userwarning might be a better choice.